stdlib/public/Concurrency/TaskPrivate.h

```diff
@@ -264,9 +264,7 @@ class alignas(sizeof(void*) * 2) ActiveTaskStatus {
 };
 
 /// The size of an allocator slab.
-///
-/// TODO: find the optimal value by experiment.
-static constexpr size_t SlabCapacity = 1024;
+static constexpr size_t SlabCapacity = 1000;
```

Comment:
The way we make this a template argument is unfortunate, because it means we have to write it out here where we can't adjust it for the slab header size, which is what we obviously should be doing instead of this magic-number adjustment.

It should probably also be smaller on 32-bit platforms.

Reply:
Maybe we could change the semantics of the template argument so that it's the requested malloc size instead of the requested slab size.

Reply:
Yeah. Let's separate out a task to make this more robust in the future.